encord/objects/coordinates.py

```diff
@@ -158,20 +158,43 @@ class SkeletonCoordinate:
     # `name` and `color` can be removed as they are part of the ontology.
     # The frontend must first be aware of how to merge with ontology.
     name: str
-    color: str
+    color: Optional[str] = None
 
     # `featureHash` and `value` seem to appear when visibility is
     # present. They might not have any meaning. Remove if confirmed that
     # Frontend does not need it.
-    feature_hash: Optional[str] = None
+    featureHash: Optional[str] = None
```

Comment:
Hm, it seems like we're changing api here? We're using snake case across the sdk (and in python in general).

Reply:
Hm, not sure I can agree here. FE is typescript, where camelCase is a default convention. While in python it's a snake case. So you can do either make this BaseDTO, and then the conversion from camelCase to snake_case will happen automatically. Or do it manually.

To reiterate: the whole point of the python sdk is to make the code that feels native in python environment. It is used by the external customers that aren't familiar with the frontend code, so discrepancy with it is not really a concern.

As to the `The SkeletonCoordinate type should ideally just be like any other coordinate` - no contradiction here. All other coordinates will eventually be BaseDTO as well. What's the point of having manual from_dict to dict methods if it all can be done automatically, with extra type safety? 
But I won't push for it too much rn, as the whole pydantic story is relatively new in sdk and might make your task harder. So if you find using dataclasses more convenient for now - go for it